testing.md

```diff
@@ -1,54 +1,50 @@
 # Testing
 
-## Unit test coverage (23rd May 2017, v0.0.0.9005)
+## Unit test coverage (7th September 2019, v0.1.2)
```

Comment:
Do we want to update this to the newest run?

Reply:
I don't think it's essential. It's just the test coverage, which won't have changed much. I'll add an item to include this next time though.